drakrun/drakrun/lib/sample_startup.py

```diff
@@ -9,53 +10,79 @@
 log = logging.getLogger(__name__)
 
 
-def get_sample_startup_command(extension: str, content: bytes) -> str:
+def get_sample_startup_command(
+    target_path: str, extension: str, entrypoints: List[str]
+) -> str:
     """Gets a startup command suitable for running the files with the provided
     extension. Sometimes content is also parsed to determine the command.
     Extension should be provided without dot, so `dll` instead of `.dll`.
     """
+    argv = get_startup_argv(target_path, extension, entrypoints)
+    return mslex.join(argv, for_cmd=False)
+
+
+def get_startup_argv(
+    target_path: str, extension: str, entrypoints: List[str]
+) -> List[str]:
     if extension == "dll":
-        return get_dll_startup_command(content)
-    if extension in ["exe", "bat"]:
-        return "%f"
-    if extension == "ps1":
-        return "powershell.exe -executionpolicy bypass -File %f"
+        if entrypoints == ["DllRegisterServer"]:
```

Comment:
shouldn't it be `"DllRegisterServer" in entrypoints`?

Reply:
Well, we don't support more than one entrypoint for DLL anyway :thinking: